### ISSUE TITLE
Tests hang when testing with scala-cli

### ISSUE THREAD

**Issue by KristianAN** [typelevel/munit-cats-effect#340]
Tests never finish when being ran with scala-cli. Same tests finish when runnning with sbt. If i make a test fail it does not hang.

Minimal case that hangs, also happens for FunSuite. If I remove the dep on munit-ce it works again.

project.scala
```scala
//> using scala "3.3.1"

//> using test.dep "org.scalameta::munit:0.7.29"
//> using dep "org.typelevel::cats-effect:3.5.2"
//> using test.dep "org.typelevel::munit-cats-effect:2.0.0-M3"
```
Tests
```
import munit.CatsEffectSuite
import cats.effect.IO

class MyTestSuite extends CatsEffectSuite:

  test("minimal"):
    IO:
      assert(true)
```

Scala CLI version: 1.0.4

**Comment by armanbilge** [comment]
Latest munit is v1.0.0-M10, please try that.

https://github.com/scalameta/munit/releases/tag/v1.0.0-M10

**Comment by KristianAN** [comment]
This fixes it.

**Comment by armanbilge** [comment]
I've released munit-cats-effect v2.0.0-M4 which brings in the latest, fixed version of munit automatically.

https://github.com/typelevel/munit-cats-effect/releases/tag/v2.0.0-M4
